compiler/rustc_codegen_ssa/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 test = false
 
 [dependencies]
-ar_archive_writer = "0.1.3"
+ar_archive_writer = "0.1.4"
```

Comment:
bumps object dependency to 0.31 https://github.com/rust-lang/ar_archive_writer/pull/6
I believe it's already resolved to 0.1.4 as Cargo.lock seems to already have it